vprof/code_heatmap.py

```diff
@@ -124,8 +125,7 @@ def run_as_package_path(self):
     def run_as_module(self):
         """Runs program as module."""
         try:
-            with open(self._run_object, 'r') as srcfile,\
-                _CodeHeatmapCalculator() as prof:
+            with open(self._run_object, 'r') as srcfile, _CodeHeatmapCalculator() as prof:
```

Comment:
I should have added `pylint` and `eslint` rules to check max line length. It should be 80 everywhere. If it's not, it was done accidentally :smile: